src/formatDate.js

```diff
@@ -7,8 +7,54 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const parts = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The assumption that the last element of `fromFormat` is the delimiter may not always be correct. Consider explicitly specifying the delimiter or handling cases where the delimiter is not the last element.